src/app/assets/logos/chains/lens.tsx

```diff
@@ -0,0 +1,39 @@
+import { SVGIconProps } from "@/types";
+
+export function Lens(props: SVGIconProps) {
+    return (
+        <svg
+            width="30.694"
+            height="30.694"
+            viewBox="0 0 30.694 30.694"
+            fill="none"
+            version="1.1"
+            id="svg1"
+            xmlns="http://www.w3.org/2000/svg"
+            {...props}
+        >
+            <rect
+                mask="url(#lens-logo-mask)"
+                width="80"
+                height="20"
+                fill="currentColor"
+                id="rect1"
+                x="0"
+                y="0"
+                transform="translate(5.012207e-5,5.3469251)"
+            />
+            <defs id="defs1">
+                <mask id="lens-logo-mask">
+                    <g fill="#ffffff" id="g1">
+                        <path
+                            fill-rule="evenodd"
```

Comment:
Let's have these props in `camelCase` to avoid console errors.

Reply:
Addressed in a5a9a872110c7a01e6e0da8c988ee4bab83bcfef